scripts/run-bundled-codegen.sh

```diff
@@ -0,0 +1,64 @@
+# Bail if errors occur
+set -e
+
+# Get the path to the script directory
+SCRIPT_DIR="$(eval dirname $0)"
+
+# Get the SHASUM of the tarball
+ZIP_FILE="${SCRIPT_DIR}/apollo.tar.gz"
+SHASUM="$(eval /usr/bin/shasum -a 256 ${ZIP_FILE})"
+SHASUM_FILE="${SCRIPT_DIR}/apollo/.shasum"
+APOLLO_DIR="${SCRIPT_DIR}"/apollo
+
+# Helper functions
+remove_existing_apollo() {
+  rm -r "${APOLLO_DIR}"
+}
+
+extract_cli() {
+  tar xvzf "${SCRIPT_DIR}"/apollo.tar.gz -C "${SCRIPT_DIR}"
+  touch "${SHASUM_FILE}"
+  tee "${SHASUM_FILE}" <<<"${SHASUM}"
+}
+
+check_shasum() {
+  # Make sure the SHASUM matches the release for this version
+  EXPECTED_SHASUM="13febaa462e56679099d81502d530e16c3ddf1c6c2db06abe3822c0ef79fb9d2  ${ZIP_FILE}"
+
+  if [ "${SHASUM}" == "${EXPECTED_SHASUM}" ]; then
+    echo "Correct version of the CLI tarball is included, checking if it's already been extracted..."
+  else
+    echo "Error: The SHASUM of this zip file does not match the official released version from Apollo! This may present security issues. Terminating code generation."
+    exit 1
+  fi
+
+  # Check if the SHASUM file has already been written for this version
+  if [ -f "${SHASUM_FILE}" ]; then
+    # The file exists, let's see if it's the same SHASUM
+    FILE_CONTENTS="$(eval cat ${SHASUM_FILE})"
```

Comment:
I love the composition of `eval cat` in terms of words 🐈, but I don't think `eval` is necessary.

Reply:
removed.